jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/list/WaitJListTests.java

```diff
@@ -0,0 +1,58 @@
+package io.github.epam.html.tests.elements.complex.list;
+
+import io.github.epam.TestsInit;
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static com.epam.jdi.light.settings.JDISettings.*;
+import static com.epam.jdi.light.settings.WebSettings.*;
+import static io.github.com.StaticSite.*;
+import static io.github.com.pages.Header.*;
+import static io.github.com.pages.SearchPage.*;
+import static io.github.epam.html.tests.site.steps.States.*;
+import static org.hamcrest.Matchers.*;
+import static org.testng.Assert.*;
+
+/**
+ * Created by Roman_Iovlev on 3/2/2018.
+ */
+public class WaitJListTests implements TestsInit {
+    @BeforeMethod
+    public void before() {
+        homePage.shouldBeOpened();
+        shouldBeLoggedIn();
+        search("jdi");
+    }
+
+    @Test
+    public void notEmptyTest() {
+        jsearchTitle.is().notEmpty();
+    }
+    @Test
+    public void notEmpty2Test() {
+        jsearchTitle.assertThat(not(empty()));
+    }
+    @Test
+    public void emptyTest() {
+        TIMEOUTS.element.setUp(2);
+        try {
+            jsearchTitle.is().empty();
+            Assert.fail("List should not be empty");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'Assert.fail' due to existing static import 'org.testng.Assert.*'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5408219)